src/ui/starter-select-ui-handler.ts

```diff
@@ -2415,7 +2415,7 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
       }
     });
 
-    this.starterSelectScrollBar.setPages(Math.ceil((this.filteredStarterContainers.length - 81) / 9) + 1);
+    this.starterSelectScrollBar.setPages(Math.max(Math.ceil(this.filteredStarterContainers.length / 9)), 1);
```

Comment:
```suggestion
    this.starterSelectScrollBar.setPages(Math.max(Math.ceil(this.filteredStarterContainers.length / 9), 1));
```
Sorry, but the parentheses in the code I just suggested were wrong. This is the correct way to fix it.

Reply:
Please be a bit more skeptical of me next time! 😄